### ISSUE TITLE
Lightning channel stuck while opening due to low fee

### ISSUE THREAD

**Issue by rkfg** [spesmilo/electrum#7306]
I'm using Electrum from master branch. I tried to open a new channel to a node and set the fee manually using the static method. Looks like it wasn't enough because I've got this in the log (there were no GUI error messages and no transactions appeared at the history tab):
```
I | transaction | adding signature for <nodeid>
D | transaction | is_complete True
I/n | network | broadcast_transaction error [DO NOT TRUST THIS MESSAGE]: RPCError(1, 'the transaction was rejected by network rules.\n\ninsufficient fee, rejecting replacement e249d9
535bef20c16b4dc3338a6d313e69b2d606f2762851942769e0a4ba5d23; new feerate 0.00010000 BTC/kvB <= old feerate 0.00010000 BTC/kvB\n[...]')
I/n | network | error: could not broadcast open_channel e249d9535bef20c16b4dc3338a6d313e69b2d606f2762851942769e0a4ba5d23, insufficient fee
Your transaction is trying to replace another one in the mempool but it does not meet the rules to do so. Try to increase the fee.
```
I don't see the funding transaction in the history and it clearly hasn't been broadcasted (this txid isn't found on any block explorers). The channel now is stuck in `REESTABLISHING` peer state, I tried to force close it (unsuccessfully, because there's no funding tx). My balance is unchanged so the only problem is this stuck channel that I can't remove. lnd has `abandonchannel` command that I believe does exactly what I want, just remove it no questions asked... but I can't find anything like that in the Electrum console or GUI.

**Comment by bitromortac** [comment]
I don't know if there's a user-facing command to abandon a channel. There's an internal method `LNChannel.remove_channel(channel_id)` that could be used.

In the console (enable View>Show Console), you could enter `wallet.lnworker.channels`, to get a list of your channels.
Then assign the channel to a variable `c = wallet.lnworker.channels[b'insert']` where `b'insert'` is the identifier from the list. It would be interesting what `c.get_state()` tells you.

**Comment by rkfg** [comment]
I did that, `c.get_state()` returns `<ChannelState.SHUTDOWN: 4>` because I tried to force close it.

**Comment by bitromortac** [comment]
Ok, I think in that case you need to manually transition the channel to the `REDEEMED` state, by calling `c.set_state(6)` (`SHUTDOWN->FORCE_CLOSING`) and then `c.set_state(8)` (`FORCE_CLOSING->REDEEMED`). Then you should be able to delete the channel in the GUI with a right click on the channel.

**Comment by rkfg** [comment]
```python
>>> c.set_state(6)
Traceback (most recent call last):
  File "/home/rkfg/soft/svn-soft/electrum/electrum/lnchannel.py", line 172, in set_state
    self.logger.debug(f'({self.get_id_for_log()}) Setting channel state: {old_state.name} -> {state.name}')
AttributeError: 'int' object has no attribute 'name'
```
I can comment this line out but would be good to fix it!

**Comment by bitromortac** [comment]
Yes, my mistake, should have done an import `from electrum.lnchannel import ChannelState` and then take the corresponding enums `ChannelState.FORCE_CLOSING` as an argument for the set_state method.

**Comment by hreppo** [comment]
I'm having a similar problem with Electrum 4.4.6. The funding transaction was rejected due to low fees. I deleted the transaction from the history tab, and now the funds are stuck in an unusable channel. The channel is in the opening state. Clicking details->funding outpoint results in in an 'transaction not found' error dialog.

I tried setting the channel to ChannelState.REDEEMED, but then I'm missing the funds in the total wallet balance.

Any help is much appreciated.
